routes/comment.js

```diff
@@ -0,0 +1,30 @@
+const router = require("express").Router();
+
+const Blog = require("../backend_model/Comment_post")
+
+
+router.get("/comment", (req, res) => {
+    res.render("/")
+})
```

Comment:
Why? Are you just testing the `get` route?

Reply:
was debugging I removed it already